R/charlson_comorbidity_index.R

```diff
@@ -194,3 +229,47 @@ charlson_comorbidity_index <- function(ipdiag, erdiag, at_admission = TRUE, raw_
 
```

Comment:
When `raw_comorbidities = FALSE` the returned table from `comorbidity_index` does not contain the comorbidity index. This naming part should only run when `raw_comorbidities == FALSE` otherwise it messes the column names in the returned table. Same issue with the `elixhauser_comorbidity_index` function.

code to reproduce the result:
```
cleandb <- dbConnect(odbc::odbc(), 
                     Driver = "PostgreSQL Unicode",
                     Server = "geminiaio.smh.smhroot.net", 
                     Port = "5432",
                     Database = "drm_cleandb_v2_1_1", 
                     UID = "guoyi", 
                     PWD = getPass::getPass("Database password: "))

erdiag <- dbGetQuery(cleandb, "select * from erdiagnosis;")
ipdiag <- dbGetQuery(cleandb, "select * from ipdiagnosis")
test <- charlson_comorbidity_index(ipdiag, erdiag, at_admission = T, raw_comorbidities = T)
```

Reply:
Great catch! Should be resolved now. Also resolved an issue where when outputting raw comorbidities, the columns were being dropped with this change.